pkg/process/analyze.go

```diff
@@ -86,10 +85,14 @@ func (a *processAnalyzer) findKeyvalMmap(pid int) (uintptr, uintptr) {
 	for _, m := range maps {
 		if m.Perms != nil && m.Perms.Read && m.Perms.Write && m.Perms.Execute {
 			log.Logger.Info("found addr of keyval map", "addr", m.StartAddr)
-			return m.StartAddr, m.EndAddr
+			return &AllocationDetails{
+				StartAddr: uint64(m.StartAddr),
+				EndAddr:   uint64(m.EndAddr),
+			}
 		}
 	}
-	panic(errors.New("cant find keyval map"))
+	log.Logger.V(1).Info("did not find any address for keyval map. Some services couldn't be instrumented")
```

Comment:
I think a better message would be something like `could not allocate remote memory, automatic context propagation will be disabled`